plugin.py

```diff
@@ -489,14 +489,17 @@ def onMQTTSubscribed(self):
                 #Domoticz.Error("onMQTTSubscribed: Error: " + str(e))
                 Domoticz.Error(traceback.format_exc())
 
+# ==========================================================DASHBOARD COMMAND=============================================================
     def onCommand(self, Unit, Command, Level, sColor):
         Domoticz.Log("onCommand " + self.deviceStr(Unit) + ": Command: '" + str(Command) + "', Level: " + str(Level) + ", Color:" + str(sColor));
 
         if Unit in Devices:
             try:
                 # TODO: Make sure the relevant command topic exists
                 configdict = json.loads(Devices[Unit].Options['config'])
-                if Command == "Set Brightness" or Command == "Set Level" or (Command == "On" and Level > 0):
+                if Command == "Set Level" and  "position_topic" in configdict:
+                    self.mqttClient.Publish(configdict["set_position_topic"],str(Level))
```

Comment:
First you check for `"position_topic"` but then you use `"set_position_topic"` when extracting the value. Typo?